recipes/geoserver-rest/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = 'geoserver-rest' %}
+{% set version = '1.4.0' %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 625327b23ee2ee77e839272b04d0b6e81b7dc98e0713e4e23d003a1bf2461551
+
+build:
+  noarch: python
+  number: 1
```

This should start from zero for each new version
```suggestion
  number: 0
```